lib/dal-test/src/schemas/test_exclusive_schema_swifty.rs

```diff
@@ -0,0 +1,132 @@
+use crate::schemas::schema_helpers::{
+    build_action_func, build_asset_func, build_resource_payload_to_value_func, create_identity_func,
+};
+use dal::pkg::import_pkg_from_pkg;
+use dal::{pkg, prop::PropPath, ActionKind, ComponentType};
+use dal::{BuiltinsResult, DalContext, PropKind};
+use si_pkg::SchemaSpecData;
+use si_pkg::{
+    ActionFuncSpec, AttrFuncInputSpec, AttrFuncInputSpecKind, PkgSpec, PropSpec, SchemaSpec,
+    SchemaVariantSpec, SchemaVariantSpecData, SiPkg, SocketSpec, SocketSpecData, SocketSpecKind,
+};
+
+pub async fn migrate_test_exclusive_schema_swifty(ctx: &DalContext) -> BuiltinsResult<()> {
+    let mut swifty_builder = PkgSpec::builder();
+
+    swifty_builder
+        .name("swifty")
+        .version("2023-05-23")
```

Comment:
nit: we could make this today. I wonder if we could ignore the date too for the future.
cc @zacharyhamm

Reply:
fixed :)